docs/modules/default-architecture/pages/loading.adoc

```diff
@@ -0,0 +1,188 @@
+= Loading
+
+Start with LangChain for loading your data instead of manually coding your own pipeline.
+The chances are good that LangChain already provides the functionalities you need.
+
+The examples use Astra {db-vector} database for the vector store and assume you have one available. If not, see xref:ROOT:prerequisites.adoc[].
+
+== Load PDF from file
+
+. Use the PyPDFLoader to load PDFs as an array of `Document` objects.
+Each `Document` object contains the page content and metadata - in this case, the page number.
++
+[source,python]
+----
+from langchain.document_loaders import PyPDFLoader
+
+loader = PyPDFLoader("example-data/example-paper.pdf")
+docs = loader.load()
+
+print(docs[0])
+----
++
+. This will print the first page of your PDF and its associated metadata, `metadata={'source': 'example-data/example-paper.pdf', 'page': 0}`.
+
+Try changing the page number in print(docs[0]) and see how the metadata tracks to your PDF pages.
+
+=== Load multiple PDFs from a directory
+
+To load a directory containing multiple pdfs, use the `PyPDFDirectoryLoader` class:
+[source,python]
+----
+from langchain.document_loaders import PyPDFDirectoryLoader
+
+loader = PyPDFDirectoryLoader("./example-data/")
+docs = loader.load()
+
+print(docs[0])
+----
+
+== Load local JSON file
+
+JSONLoader uses the jq library to parse JSON files. If you're familiar with jq, you'll probably find this loader very familiar.
```

Comment:
ticks around JSONLoader and jq?

This might be personal preference. But I see you're doing it in other areas too, so suggestion it for all